test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTests.cs

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if IS_DESKTOP
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Common;
+using NuGet.Packaging.Signing;
+using NuGet.Test.Utility;
+using Test.Utility.Signing;
+using Xunit;
+
+namespace NuGet.Packaging.FuncTest
+{
+    [Collection(SigningTestCollection.Name)]
+    public class SignatureTests
+    {
+        private SigningSpecifications _specification => SigningSpecifications.V1;
+
+        private SigningTestFixture _testFixture;
+        private TrustedTestCert<TestCertificate> _trustedTestCert;
+        private TestCertificate _untrustedTestCertificate;
+
+        public SignatureTests(SigningTestFixture fixture)
+        {
+            _testFixture = fixture ?? throw new ArgumentNullException(nameof(fixture));
+            _trustedTestCert = _testFixture.TrustedTestCertificate;
+            _untrustedTestCertificate = _testFixture.UntrustedTestCertificate;
+        }
+
+
+        [CIOnlyFact]
+        public async Task Verify_WithUntrustedSelfSignedCertificateAndNotAllowUntrustedSelfSignedCertificate_FailsAsync()
```

Comment:
The last "SelfSignedCertificate" should be "Root".  The feedback applies to other tests as well.